src/content.ts

```diff
@@ -1434,7 +1435,7 @@ function updatePreviewBar(): void {
                 showLarger: segment.actionType === ActionType.Poi,
                 description: segment.description,
                 source: segment.source,
-                requiredSegment: hashParams.requiredSegment && segment.UUID === hashParams.requiredSegment
+                requiredSegment: requiredSegment && segment.UUID === requiredSegment || segment.UUID.startsWith(requiredSegment)
```

Comment:
```suggestion
                requiredSegment: requiredSegment && segment.UUID.startsWith(requiredSegment)
```

Reply:
would the `===` comparison not be faster? or at that point would it be better to just omit first and startWith()

Reply:
yea I guess. Then can you add brackets to make operator presence clear?